### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TheStigh**
Complete **/installerscript** folder with examples and scripts for installation on a clean RPi

### CODE REVIEW

**Comment by TheStigh** [comment]
Here you go Odi 👍

**Comment by TheStigh** [comment]
Done

File: installerscript/appdaemon.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      token: <your long-lived access token created in HA>
+      ### You must create a long-lived token in HA for AppDaemon to be used here
+
+    MQTT: 
+      type: mqtt
+      namespace: mqtt
+      client_host: <ip address to your MQTT broker>
+      client_user: <username to your MQTT broker>
+      client_password: <password to your MQTT broker>
+      client_topics:
```
Comment (Odianosen25) [inline installerscript/appdaemon.yaml]: We can leave out the client topics, as AD will subscribe to all topics on the broker by default

Comment (TheStigh) [reply installerscript/appdaemon.yaml]: Fixed

**Comment by TheStigh** [comment]
Updated README and appdaemon.yaml template

**Comment by TheStigh** [comment]
Made installerscripts more effective with less interaction

**Pull request merged**